src/assets/honey.svg

```diff
@@ -1 +1 @@
F"/><path d="M17.17-8.034l-6.357-3.641a2.438 2.438 0 00-2.444.008L2.035-7.983A2.451 2.451 0 00.819-5.865l.025 7.329a2.455 2.455 0 001.23 2.114l6.357 3.64a2.439 2.439 0 002.445-.007l6.336-3.685a2.46 2.46 0 001.21-2.12l-.025-7.33a2.456 2.456 0 00-1.226-2.11zm29.27 2.217l-3.93-2.25a1.507 1.507 0 00-1.51.005l-3.914 2.277a1.516 1.516 0 00-.752 1.309l.016 4.53a1.517 1.517 0 00.76 1.306l3.929 2.25a1.508 1.508 0 001.511-.005l3.916-2.278a1.52 1.52 0 00.748-1.31l-.015-4.53a1.518 1.518 0 00-.76-1.304z" fill="#FFDD0F"/><path d="M21.739-11.513l-10.287-5.89a3.943 3.943 0 00-3.954.012L-2.75-11.43a3.966 3.966 0 00-1.967 3.425l.04 11.86a3.972 3.972 0 001.992 3.419l10.284 5.889a3.947 3.947 0 003.957-.011l10.252-5.963a3.979 3.979 0 001.957-3.431l-.04-11.859a3.976 3.976 0 00-1.985-3.412zM53.99 44.835l-10.287-5.89a3.943 3.943 0 00-3.954.012l-10.25 5.961a3.967 3.967 0 00-1.966 3.426l.04 11.859a3.972 3.972 0 001.992 3.42l10.284 5.889a3.947 3.947 0 003.957-.011l10.252-5.964a3.979 3.979 0 001.957-3.43l-.04-11.86a3.977 3.977 0 00-1.985-3.412z" fill="#FFF19F"/><path d="M49.324 48.35l-6.358-3.64a2.438 2.438 0 00-2.444.007l-6.334 3.685a2.451 2.451 0 00-1.216 2.117l.025 7.33a2.456 2.456 0 001.23 2.113l6.357 3.64a2.439 2.439 0 002.445-.007l6.336-3.686a2.46 2.46 0 001.21-2.12l-.025-7.33a2.457 2.457 0 00-1.226-2.11z" fill="#FFEB6F"/><path d="M17.133 48.423l-6.358-3.64a2.438 2.438 0 00-2.443.008l-6.335 3.684a2.452 2.452 0 00-1.216 2.118l.025 7.329a2.454 2.454 0 001.231 2.114l6.356 3.64a2.438 2.438 0 002.446-.008l6.336-3.685a2.46 2.46 0 001.21-2.12l-.025-7.33a2.457 2.457 0 00-1.227-2.11zM46.44 50.64l-3.93-2.25a1.507 1.507 0 00-1.51.005l-3.914 2.277a1.515 1.515 0 00-.752 1.309l.016 4.53a1.517 1.517 0 00.76 1.306l3.929 2.25a1.508 1.508 0 001.511-.005l3.916-2.277a1.52 1.52 0 00.748-1.311l-.015-4.53a1.518 1.518 0 00-.76-1.304z" fill="#FFDD0F"/><path d="M21.8 44.944l-10.288-5.89a3.943 3.943 0 00-3.953.012l-10.25 5.962a3.967 3.967 0 00-1.967 3.425l.04 11.86a3.973 3.973 0 001.993 3.42L7.659 69.62a3.948 3.948 0 003.956-.011l10.252-5.963a3.978 3.978 0 001.957-3.431l-.04-11.859a3.974 3.974 0 00-1.985-3.413z" fill="#FFF19F"/><path d="M1.286-1.487l-6.359-3.64a2.438 2.438 0 00-2.444.007l-6.334 3.685A2.452 2.452 0 00-15.067.682l.025 7.33a2.454 2.454 0 001.231 2.114l6.356 3.64a2.44 2.44 0 002.445-.008l6.337-3.685a2.46 2.46 0 001.21-2.12L2.511.622a2.457 2.457 0 00-1.226-2.11z" fill="#FFDD0F"/><path d="M17.05-8.034l-6.357-3.641a2.438 2.438 0 00-2.444.008L1.915-7.983A2.451 2.451 0 00.699-5.865l.025 7.329a2.455 2.455 0 001.23 2.114l6.357 3.64a2.439 2.439 0 002.445-.007l6.337-3.685a2.46 2.46 0 001.21-2.12l-.026-7.33a2.456 2.456 0 00-1.226-2.11zm0 56.31l-6.357-3.64a2.438 2.438 0 00-2.444.008l-6.334 3.684a2.451 2.451 0 00-1.216 2.118l.025 7.329a2.455 2.455 0 001.23 2.114l6.357 3.64a2.44 2.44 0 002.445-.007l6.337-3.685a2.46 2.46 0 001.21-2.121l-.026-7.33a2.456 2.456 0 00-1.226-2.11z" fill="#FFEB6F"/></g><defs><clipPath id="clip0"><rect width="52" height="52" rx="26" fill="#fff"/></clipPath></defs></svg>
```

Comment:
i think that we are sending again the same two svg? or we optimized them?

Reply:
Ah no, we just optimized them. Was my fault for not optimizing only the one added.